lib/stat_tracker.rb

```diff
@@ -1,84 +1,89 @@
 require_relative './games_collection'
 require_relative './teams_collection'
 require_relative './game_teams_collection'
+require 'csv'
 
+#THE PARENT
 class StatTracker
-  attr_reader :games_collection, :teams_collection, :game_teams_collection
+  attr_reader :games, :teams, :game_teams
 
-  def initialize(games_collection, teams_collection, game_teams_collection)
-    @games_collection = games_collection
-    @teams_collection = teams_collection
-    @game_teams_collection = game_teams_collection
+  def initialize(locations)
+    @games = GamesCollection.new(locations[:games], self)
```

Comment:
This is definitely necessary since we need to be able to call the Collections directly in order to make our "pants" function correctly